nextflow.config

```diff
@@ -67,6 +67,15 @@ profiles {
     params.challenge_container = "ghcr.io/sage-bionetworks-challenges/pegs-evaluation:latest"
     params.send_email = true
     params.email_with_score = "no"
+  }
+  olfactory25_challenge_validate {
+    params.entry = 'data_to_model'
+    params.view_id = "syn66279193"
```

Comment:
This view ID will be the ID of your submissions view, which is where the workflow pulls new submissions from (based on the `status=RECEIVED` annotation)